src/shared/storage/index.js

```diff
@@ -22,9 +22,72 @@ class Transaction {
     }
 }
 
+class Node {
+    static schema = NodeSchema;
+
+    /**
+     * Returns a list of nodes
+     * @return {Realm.Results}
+     */
+    static getNodes() {
+        return realm.objects('Node');
+    }
+
+    /**
+     * Returns the list of nodes that support remote PoW
+     * @return {Realm.Results}
+     */
+    static getRemotePowNodes() {
+        return this.getNodes().filtered('remotePow == true');
+    }
+
+    /**
+     * Adds a custom node
+     * @param {string} url Node URL
+     */
+    static addCustomNode(url) {
+        realm.write(() => {
+            realm.create('Node', {
+                url: url,
+                custom: true,
+                // TODO: Where should we check for remote PoW support?
```

Comment:
This will be helpful https://github.com/iotaledger/iri/issues/1039

Till then, we can just make a dummy `attachToTangle` request.